base/rational.jl

```diff
@@ -49,6 +49,11 @@ Rational(n::T, d::T) where {T<:Integer} = Rational{T}(n, d)
 Rational(n::Integer, d::Integer) = Rational(promote(n, d)...)
 Rational(n::Integer) = unsafe_rational(n, one(n))
 
+"""
+    divgcd(x, y)
+
+Returns `(x÷gcd(x,y), y÷gcd(x,y))`. See also [`gcd`](@ref).
+"""
```

Comment:
```suggestion
"""
    divgcd(x::Integer, y::Integer)

Returns `(x÷gcd(x,y), y÷gcd(x,y))`.

See also [`div`](@ref), [`gcd`](@ref).
"""
```

It's good to note the type restriction in the docstring.